proof_test.go

```diff
@@ -893,3 +894,53 @@ func TestVerifyNamespace_ShortAbsenceProof_Invalid(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoToProof_InclusionProof(t *testing.T) {
```

Comment:
[Optional but highly recommended] In order to align with the testing style in the nmt, would it be possible for you to modify your unit test to follow an approach similar to [this](https://github.com/celestiaorg/nmt/blob/3693c9a626b7ddaae996b3c75602f5accfa45739/proof_test.go#L103)? This convention involves creating a slice of test cases using a struct that defines the input/output parameters for each test case. Each test case represents a specific scenario or condition that the function being tested should satisfy.

Reply:
Yes. Sure. Will do first thing tomorrow morning.
Thank you for the review.